spec/cohort.spec.js

```diff
@@ -0,0 +1,208 @@
+import { Cohort } from '../src/cohort.js'
+import { StudentManager } from '../src/student-manager.js'
+import { Student } from '../src/student.js'
+import { CohortManager } from '../src/cohort-manager.js'
+
+describe('cohort', () => {
+  let studentManager
+  let cohortManager
+  let student1
+  let student2
+  let student3
+  let student4
+  beforeEach(() => {
+    studentManager = new StudentManager()
+    cohortManager = new CohortManager()
+    student1 = new Student(
+      'Lee',
+      'Smith',
+      'koala333',
+      'lee.smith@hotmail.co.uk'
+    )
+    student2 = new Student('Jen', 'Smith', 'panda93', 'jen.smith@gmail.com')
+    student3 = new Student(
+      'Matt',
+      'Micheal',
+      'random203956',
+      'm.michael@gmail.com'
+    )
+    student4 = new Student('Matt', 'Smith', 'citizenErased', 'm.kerr@gmail.com')
+    studentManager.handleNewItem(student1)
+    studentManager.handleNewItem(student2)
+    studentManager.handleNewItem(student3)
+    studentManager.handleNewItem(student4)
```

Comment:
If only there were a way to have some sort of a list of items and a way to programmatically repeat an operation 🤔